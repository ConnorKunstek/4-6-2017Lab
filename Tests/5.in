DDDSDDSD
DDDSDDSD
DDDSDDSD
DFDSDDSD
DDDSDDSD
DDDSDDSS
DDDSDDSD
4 2
1 1
7 1
