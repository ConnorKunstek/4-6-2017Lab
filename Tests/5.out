Yes
Yes
Yes
Time: 0.001099 seconds

